src/main/java/io/apimatic/coreinterfaces/authentication/Authentication.java

```diff
@@ -15,4 +15,9 @@ public interface Authentication {
      * @return the authenticated request
      */
     Request apply(Request httpRequest);
+    
+    /**
+     * Validate the authentication on the httpRequest
```

Comment:
Description should look something like "validates the auth params for the Http Request."

Reply:
The comment has been updated